PDKTSwipableCard/CardBehaviour.swift

```diff
@@ -58,14 +58,14 @@ class CardBehaviour : UIDynamicBehavior {
         }
     }
     
-    override func willMoveToAnimator(dynamicAnimator: UIDynamicAnimator?) {
-        super.willMoveToAnimator(dynamicAnimator)
+	override func willMove(to dynamicAnimator: UIDynamicAnimator?) {
+		super.willMove(to: dynamicAnimator)
```

Comment:
It seems that you are using tabs over spaces. I don't have a strong opinion about this, but it would be great to keep 4 spaces for consistency in this project. 

Sorry for this style nazi comment 😓

Reply:
No worries 😅 I'm used to changing my indentation style for my OSS work (I've always used the defaults for some reason, maybe I should change 🧐